include/dsAudio.h

```diff
@@ -2063,6 +2063,30 @@ dsError_t  dsGetSecondaryLanguage(intptr_t handle, char* sLang);
  */
 dsError_t dsGetHDMIARCPortId(int *portId);
 
+/**
+* @brief Sets the Mixer Volume level for the MS12 input
+*
+* This function sets the MS12 volume level for either primary(main audio) or system audio input(Text to Speech/System Beep)
+*
+* @param[in] handle  - Handle for the output Audio port
+* @param[in] aInput  - dsAudioInputPrimary / dsAudioInputSystem. Please refer ::dsAudioInput_t
+* @param[in] volume  - volume to be set (0 to 100)
+*
+* @return dsError_t                        - Status
+* @retval dsERR_NONE                       - Success
+* @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+* @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+* @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported; e.g: source devices
+* @retval dsERR_OPERATION_FAILED           - The attempted operation has failed
```

Comment:
Why would the operation fail? Specifically, what reason are you suggesting here why it wouldnt' work?

Reply:
Made it dsERR_GENERAL